webpack/ForemanTasks/Components/TasksTable/index.js

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import TasksTable from './TasksTable';
+import reducer from './TasksTableReducer';
+import * as actions from './TasksTableActions';
+import {
+  selectResults,
+  selectTasksTable,
+  selectQuery,
+  selectPagitation,
+  selectItemCount,
+  selectLoading,
+} from './TasksTableSelectors';
+
+const mapStateToProps = state => ({
+  ...selectTasksTable(state),
```

Comment:
IMO it is risky to use `...` from a selector, consider moving the fields here and create a selector for each.